packages/turbine-js-cli/src/runner/primary.ts

```diff
@@ -52,6 +127,34 @@ export default class Primary extends Base {
     }
   }
 
+  async runAppPlatformV2(
+    functionImageName: string,
+    appName: string,
+    headCommit: string,
+    version: string,
+    spec: string
+  ) {
+    const environment = new PlatformV2Runtime(
+      functionImageName,
+      appName,
+      headCommit,
+      version,
+      spec
+    );
+
+    try {
+      await this.dataApp.run(environment);
+      const IR = environment.serializeToIR();
+      console.log(IR);
+      // POST IR
+      return Ok(true);
+    } catch (e) {
+      assertIsError(e);
+      console.log(e);
+      return Err(new BaseError("Error listing resources", e));
```

Comment:
What do you think about making this error message more generic for any cases in which the run time fails, e.g.:

```suggestion
      return Err(new BaseError("Error running app in v2 mode", e));
```